src/parsers/perf/perfparser.cpp

```diff
@@ -74,18 +74,35 @@ struct StringId
     qint32 id = -1;
 };
 
+struct StringId64
```

Comment:
I don't understand why this is needed - is the string id suddenly 64bit? if so, then the non-64 string id needs to be removed _everywhere_

Reply:
but, more probably, this indicates a bug in perfparser that now writes 64bit string ids somewhere? these should remain at 32bit

Reply:
ah, looking down I realize where this comes from, remove this class please and use `quint64` directly for the addresses below